archetypes/Exchange/TokenSelect/TokenSelect.tsx

```diff
@@ -79,7 +79,7 @@ const TokenSelect: React.FC<{
                                 >
                                     <Styles.TokenSelectCell hasLogo>
                                         <Styles.TokenLogo size="md" ticker={tokenSymbolToLogoTicker(token.symbol)} />
-                                        {token.symbol}+{token.pool.settlementTokenSymbol}-01
+                                        {token.symbol}-01
```

Comment:
whats the origin of this `-01` on the end?

Reply:
no idea, was just something they wanted in the designs